Second milestone for portfolio page

frnega opened the pull request:
this is my portfolio page changed the template in to my format

zieeco reviewed (changes requested):
Hi @<STUDENT NAME>,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


<img src="https://media0.giphy.com/media/8UGVQESpDMMQl7yd3F/giphy.gif"/>


## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

frnega commented:
changed the headline to section ,added acknowledgement and  getting started on readme file  and also added flexbox a
also increased the size of hamburger

vechicin commented on index.html:
```diff
@@ -1,33 +1,42 @@
 <!DOCTYPE html>
 <html>
-	<head class="main">
+	<head >
```
It seems to me you misunderstood the previous reviewer's requirements. Your page is now missing a lot of content from the template design.

<img width="199" alt="Screen Shot 2022-08-22 at 8 04 01" src="https://user-images.githubusercontent.com/82426682/185927983-b2d92b8f-86ea-4b16-b4be-809c9902f05c.png">

<img width="246" alt="Screen Shot 2022-08-22 at 8 04 18" src="https://user-images.githubusercontent.com/82426682/185928039-d677ff0f-5ced-486e-af3c-9535e92a951a.png">

Please add all of the content back, and make sure you align it with flexbox, make sure you match the margins/paddings from the template design, make sure you use the same colors and fonts as well and make sure the background images and icons are correctly sized and positioned.

vechicin reviewed (changes requested):
# Hi @frnega ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Good job fixing the README file

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

frnega commented:
tried to change as expected what i wanted to be in my future portfolio
 but the template  is deprecated  but tried all the best i can

Bondok6 reviewed (approved):
# STATUS: APPROVED :green_circle: :clap: 

## Hi @frnega 🙋‍♂️, 

I appreciate that you implemented the changes requested by previous reviewers!
Please pay attention to the social icons section, as now it does not look like a copy of the Figma design.

But for the hard work you've done, your project deserves to be approved, and I am sure that you will fix this minor issue.

Your project is complete!  Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your repo is professional ✅
- Well structured files ✅
- No linter errors ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

<img src="https://camo.githubusercontent.com/54a5d803a138b5a1a1242a607e4c1192c852cab8c591c1ddcaa33d06b51560f6/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f686c516559335062684833692f67697068792e676966" />


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
